### PR TITLE
BaSP-MR-11:  DELETE,POST endpoints, validations and postman documentation for Admins.

### PR DESCRIPTION
**Pull request by fernandofernandezuy**
I worked on creating the Model of Admin, DELETE (for remove and admin) and POST (for creating an admin) endpoints using moongose to make the request to our database in MongoDB. Also, I validated with Joi the admin creation using a middleware function in the route and added the postman collection for all the endpoints.

### CODE REVIEW

**Review by NicolasKler — APPROVED** [review]
Looks good to me

**Review by GaloDurante — APPROVED** [review]
Looks good!

File: src/controllers/admins.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        message: 'Error',
+        error,
+      });
+    });
+};
+
+const deleteAdmin = (req, res) => {
+  const { id } = req.params;
+  if (!id.match(regexObjectId)) {
+    res.status(400).json({
```
Comment (Juanoauil) [inline src/controllers/admins.js:37]: you should return this

File: src/controllers/admins.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      message: 'Please put a valid ID',
+      data: undefined,
+      error: true,
+    });
+  }
+  Admin.findByIdAndDelete(id)
+    .then((admin) => {
+      if (!admin) {
+        res.status(404).json({
```
Comment (Juanoauil) [inline src/controllers/admins.js:46]: And this too

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**